### PR TITLE
Add 4 new environments: demo, unity, clever, burn

### PR DESCRIPTION
**Pull request by sunny-viktoryia**
Add the config settings for 4 new firebase environments.

partially closes: https://github.com/sparkletown/internal-sparkle-issues/issues/731
partially closes: https://github.com/sparkletown/internal-sparkle-issues/issues/780
partially closes: https://github.com/sparkletown/internal-sparkle-issues/issues/782
partially closes: https://github.com/sparkletown/internal-sparkle-issues/issues/784

### CODE REVIEW

**Review by mike-lvov — APPROVED** [review]
LGTM

File: .firebaserc [hunk 1]
```diff
@@ -20,7 +20,11 @@
     "memrise": "sparkle-memrise",
     "unesco": "sparkle-unesco",
     "ohbm": "sparkle-ohbm",
-    "sparkle-pa": "sparkle-pa"
+    "sparkle-pa": "sparkle-pa",
+    "sparkle-demo": "sparkle-demo-e65b1",
```
Comment (0xdevalias) [inline .firebaserc:24]: Curious, why did the demo environment end up with this suffix on the projectId?

File: .firebaserc [hunk 2]
```diff
@@ -175,6 +179,34 @@
           "sparkle-pa"
         ]
       }
+    },
+    "sparkle-demo": {
```
Comment (0xdevalias) [inline .firebaserc:183]: Not 100% but I think this chunk may be wrong based on the demo projectId

File: firebase.json [hunk 3]
```diff
@@ -237,6 +237,50 @@
         }
       ],
       "ignore": ["firebase.json", "**/.*", "**/node_modules/**"]
+    },
+    {
+      "target": "sparkle-demo",
```
Comment (0xdevalias) [inline firebase.json:242]: Does this work correctly given it differs from the demo projectId?

**Pull request merged**